internal/gapicgen/gensnippets/gensnippets.go

```diff
@@ -268,4 +268,6 @@ const licenseHeader string = `// Copyright %v Google LLC
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Code generated by cloud.google.com/go/internal/gapicgen/gensnippets. DO NOT EDIT.
```

Comment:
We could exclude the `by cloud.google.com/go/internal/gapicgen/gensnippets` altogether per the documented rule.